src/main.js

```diff
@@ -15,10 +15,7 @@ let el = document.createElement('div')
 el.id = 'firstrunwizard'
 document.querySelector('body').appendChild(el)
 
-const app = new Vue({
-	el: '#firstrunwizard',
-	name: 'FirstRunWizardRoot',
-	render: h => h(App)
-})
+const View = Vue.extend(App)
+const vm = new View().$mount(el)
 
-window.OCA.FirstRunWizard = app.$children[0]
+window.OCA.FirstRunWizard = vm
```

Comment:
Debug only?

Reply:
No, the open method will be called for activation and when clicking the about menu entry

Reply:
Ooooh, that is one way to do it ^^
I would have only exposed the open function maybe or something for safety ;)

Reply:
Sure, will do that ;)